apache2/msc_logging.c

```diff
@@ -1,2285 +1,2288 @@
-/*
```

Comment:
This seems the \r\n problem.
I explicitely replaced all \r\n by \r but git can't see any diff to commit.
What should I do?